terms/neuro/climbID.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://repo-prod.prod.sagebase.org/repo/v1/schema/type/registered/sage.annotations-neuro.climbID-0.0.1",
+  "description": "Identifying string linked to an animal in Climb database",
```

Comment:
Can we specify where the Climb database is? (i.e. hosted by Jackson Laboratories)

Reply:
I think each lab manages its own instance of Climb. We can include a URL to the developer if helpful. https://www.rockstepsolutions.com

Reply:
Ah, never knew that. Yeah, let's include the link to the developer.